stl/inc/filesystem

```diff
@@ -921,7 +921,7 @@ namespace filesystem {
             return operator/=(_Replacement);
         }
 
-        path& replace_extension(/* const path& _Replacement = path() */) noexcept /* strengthened */ {
+        path& replace_extension(/* const path& _Replacement = path{} */) noexcept /* strengthened */ {

```

Comment:
I think these occurrences could all be simply `{}`. Should they be?

Reply:
I thought about that, but since `const path&` is binding to a temporary, I wondered whether being a little more verbose was a good idea. (I did try it for `file_status` passed by value, but the experimental one has an explicit default constructor, hssk.) I can make this change,